data/human/free worlds intro.txt

```diff
@@ -12,51 +12,60 @@ mission "Pact Recon 0"
 	minor
 	name "Pirate Reconnaissance"
 	description "Fly through the <waypoints> systems, then report to <destination> so the Southern Defense Pact can analyze your logs of pirate activity (if any)."
-	waypoint "Alniyat"
-	waypoint "Han"
-	waypoint "Atria"
-	waypoint "Lesath"
 	source
-		attributes "south"
+		attributes "rim" "south"
 		not government "Pirate"
+		not system "Alniyat" "Atria" "Lesath" "Han"
 	destination "Glaze"
+	waypoint "Alniyat"
+	waypoint "Atria"
+	waypoint "Han"
+	waypoint "Lesath"
 	to offer
 		random < 60
 		not "event: war begins"
 	
 	on offer
 		conversation
-			`You can't help noticing that the freighter parked next to the <ship> is in very bad shape, its hull pockmarked with blaster scars, as well as larger dents that may be from missiles. The crew is busy welding new hull plating onto it; the captain is having an animated conversation with a young female militia officer.`
+			`You can't help noticing that the freighter parked next to the <ship> is in very bad shape, its hull pockmarked with blaster scars, as well as larger dents that may be from missiles. The crew and a number of men in militia uniforms are busy welding new hull plating onto it; the captain is having an animated conversation with a young female militia officer.`
 			choice
 				`	(Approach them and find out what happened.)`
 				`	(Ignore them.)`
 					decline
-			`	You overhear enough of the conversation to figure out that the freighter was attacked by pirates. Eventually, they stop talking, and the militia officer, seeing you nearby, walks over to you. "The pirate fleets keep getting stronger every year," she says.`
+			`	You overhear enough of the conversation to figure out that the freighter was attacked by pirates, and the militia are helping to get the freighter in working condition again. "A little spit shine and it'll look as good as new," the militia officer says, but the captain of the ship doesn't look convinced.`
+			`	The militia officer glances over her shoulder and, having spotted you listening in, waves you over. "No need to be shy," she says. "I'm guessing you're not from around here."`
```

Comment:
```suggestion
			`	The officer glances over her shoulder and, having spotted you listening in, waves you over. "No need to be shy," she says. "I'm guessing you're not from around here."`
```
Repetitive

Reply:
I'd probably take out the second instance of "militia" in the first paragraph, rather than the only instance of "militia" in the second. Spread the militia's out a little bit.